src/Components/VendorCard/VendorCard.scss

```diff
@@ -4,11 +4,15 @@
   align-items: center;
   justify-content: space-around;
   margin-bottom: 35px;
+  letter-spacing: 1px;
   height: 420px;
   width: 400px;
-  border: 3px solid black;
+  border: 3px solid khaki;
   border-radius: 10px;
-  background-color: azure;
+  background-color: rgba(0,0,0,0.3);
```

Comment:
I think that these backgrounds need to be darker. like .7 or .8 - the background image is pretty busy with a lot going on and with that showing through the text becomes hard to read